sdk_entrepot_gpf/scripts/entities.py

```diff
@@ -157,28 +184,57 @@ def action_upload_delete_files(upload: Upload, delete_files: List[str]) -> None:
 
         Config().om.info(f"Suppression de {len(delete_files)} fichiers téléversés sur la livraison {upload} :")
         upload.api_open()
-        for file in delete_files:
-            if(file.__contains__(".md5")):
-                upload.api_delete_md5_file(file)
+        for s_file in delete_files:
+            if ".md5" in s_file:
+                upload.api_delete_md5_file(s_file)
             else:
-                upload.api_delete_data_file(file)
+                upload.api_delete_data_file(s_file)
 
     @staticmethod
     def action_upload_delete_failed_files(upload: Upload) -> None:
+        """
+        Supprime les fichiers mal téléversés sur la livraison
+
+        Args:
+            Upload: livraison
+        """
         Config().om.info(f"Suppression des fichiers mal téléversés sur la livraison {upload} :")
-        files = []
-        checks = upload.api_list_checks()
-        for verification in checks["failed"]:
-            check = CheckExecution(verification)
-            lines = check.api_logs_filter("ERROR")
-            for line in lines:
-                correspondance = re.search(r'\((.*?)\)', line)
-                if correspondance:
-                    Config().om.warning(correspondance.group(1))
-                    files.append(correspondance.group(1))
+        l_files = []
+        l_checks = upload.api_list_checks()
+        for d_verification in l_checks["failed"]:
+            o_check = CheckExecution(d_verification)
+            l_lines = o_check.api_logs_pages_filter(str_filter="ERROR")
+            for s_line in l_lines:
+                d__correspondance = re.search(r"\((.*?)\)", s_line)
+                if d__correspondance:
+                    Config().om.warning(d__correspondance.group(1))
+                    l_files.append(d__correspondance.group(1))
         upload.api_open()
-        for file in files:
-            upload.api_delete_data_file(file)
+        for s_file in l_files:
+            upload.api_delete_data_file(s_file)
+
+    @staticmethod
+    def action_execution_logs(verification: LogsInterface, filters: str) -> None:
+        """
+        Applique les filtres au logs de la verification
+        Args:
+            verification: LA verification ou vont être appliqué les filtres
+            filters: Les différents filtres qui seront appliqués sur la verification
+        """
+        Config().om.info(f"Affichage des logs sur la verification {verification}")
+
+        o_pattern = r"(\-?\d+)(?::(\-?\d+))?(?:/(\-?\d+))?\|?(\w*)?"
+        o_match = re.match(o_pattern, filters)
+        i_firstpage, i_lastpage, i_lineperpage, s_filter = o_match.groups()
+        if i_lastpage is None:
+            i_lastpage = 0
+        if i_lineperpage is None:
+            i_lineperpage = 1000
```

Comment:
```suggestion
            i_lineperpage = 2000
```

on peut en demander 2000, autant faire ça et faire 2 fois moins de requêtes